iphone/Maps/Classes/MapViewController.mm

```diff
@@ -934,6 +939,10 @@ - (void)zoomIn {
   GetFramework().Scale(Framework::SCALE_MAG, true);
 }
 
+- (void)reCenterMap {
+  GetFramework().SwitchMyPositionNextMode();
```

Comment:
Are you sure it's the right method to center the map? I think there was another one.

Reply:
That's the only one I found to be honest, it uses the same function as the "arrow" button on the right side of the map to switch between North/True Heading. If there is a dedicated zoom to default and center function then by all means let me know so I can update it.

Reply:
Your current implementation will switch between north/true heading if the position is already at the center. It's not too intuitive.
By 'center' you mean restore north => up after map rotation, right?
Which "default" zoom do you want this hotkey to follow? The current implementation does not change the zoom level.